modin/experimental/cloud/base.py

```diff
@@ -26,6 +26,8 @@ def __init__(self, *args, cause: BaseException = None, traceback: str = None, **
         self.traceback = traceback
         super().__init__(*args, **kw)
 
+    def __str__(self):
+        return f"\ncause: {self.cause}\n" + super().__str__()
```

Comment:
```suggestion
        if self.clause:
            return f"clause: {self.cause}\n{super()}"
        return str(super())
```

IMHO this would produce a nicer error, plus handles `self.cause` being `None`.